submodules-tasks.yml

```
@@ -0,0 +1,364 @@
+version: "3"
+vars:
+    G_DEP_DOWNLOAD_SCRIPT: "{{.ROOT_DIR}}/tools/scripts/deps-download/download-dep.py"
+
+tasks:
+  default:
+    cmds:
```

```suggestion
    deps:
```

To enable parallel execution.